integration-libs/cdc/user-profile/update-email/cdc-update-email-component.service.ts

```diff
@@ -13,11 +14,17 @@ import {
   GlobalMessageType,
   RoutingService,
 } from '@spartacus/core';
-import { UpdateEmailComponentService } from '@spartacus/user/profile/components';
+import {
+  USE_MY_ACCOUNT_V2_EMAIL,
+  UpdateEmailComponentService,
+} from '@spartacus/user/profile/components';
 import { UserEmailFacade } from '@spartacus/user/profile/root';
+import { Subject } from 'rxjs';
 
 @Injectable()
 export class CDCUpdateEmailComponentService extends UpdateEmailComponentService {
+  updateSucceed$ = new Subject();
+  enableMyAccountV2 = inject(USE_MY_ACCOUNT_V2_EMAIL);
```

Comment:
[MEDIUM]
When injecting v2 feature flags, please let's make them `private` properties, not to pollute the Public API surface of the component, and so we're able to remove this property in the future without a breaking change.

Note: similar remark applies to also other places in this PR, where we inject a feature flag and assign to a class' property.

Reply:
reverted